base/boot.jl

```diff
@@ -965,4 +966,7 @@ arraysize(a::Array) = a.size
 arraysize(a::Array, i::Int) = sle_int(i, nfields(a.size)) ? getfield(a.size, i) : 1
 export arrayref, arrayset, arraysize, const_arrayref
 
+# For convenience
+EnterNode(old::EnterNode, new_dest::Int) = EnterNode(new_dest)
```

Comment:
This method probably looks confusing here, but it helps minimize the diff when extra fields are added to EnterNode, because a lot of places just need to rename the dest without touching the rest of the fields.